Makefile

```diff
@@ -1,7 +1,7 @@
 TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 RELEASE_VERSION ?= latest
 
-SUBDIRS=docker-images certificate-manager crd-generator api cluster-operator topic-operator kafka-init examples
+SUBDIRS=docker-images api certificate-manager crd-generator cluster-operator topic-operator kafka-init examples
```

Comment:
Actually ... the `api` is dependent on `crd-generator`. So what we need here is not to move `api` to the front but to move the `certificate-manager` after `api`. I.e. we need this:

```
SUBDIRS=docker-images  crd-generator api certificate-manager cluster-operator topic-operator kafka-init examples
```

Reply:
Done